add.php

```diff
@@ -2,6 +2,38 @@
 
 require_once "functions.php";
 
-$db = getGuitarDatabase();
-addGuitarToDatabase($db, $_POST);
-var_dump($_POST);
\ No newline at end of file
+if (empty($_POST["brand"])){
```

Comment:
I can't remember if there should be a space between ) and { for if statements or not...?